webservices/rest/src/main/java/com/epam/rest/Application.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.rest;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan
+@EnableAutoConfiguration
```

Comment:
Actually `@SpringBootApplication` [includes both of rest annotations](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/SpringBootApplication.java#L51) :)